docs/_docs/httpbody.md

```diff
@@ -0,0 +1,43 @@
+---
+category: documentation
+---
+
+# HttpBody message Feature
+The [HTTP Body](https://github.com/googleapis/googleapis/blob/master/google/api/httpbody.proto) messages allows a response message to be specified with custom data content and a custom content type header. The values included in the HTTPBody response will be used verbatim in the returned message from the gateway. Make sure you format your response carefully!
+
+## Example Usage
+1. Create a mux with the HTTP Body Marshaler as option. 
+
+```golang 
+	mux := runtime.NewServeMux(runtime.SetHTTPBodyMarshaler)
+```
+2. Define your service in gRPC with an httpbody response message
+
+```golang
+import "google/api/httpbody.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+service HttpBodyExampleService {
+
+ 	rpc HelloWorld(google.protobuf.Empty) returns (google.api.HttpBody) {
+        option (google.api.http) = {
+            get: "/helloworld"
+        };
+	}	
+
+}
```

Comment:
Something happened to the indentation here.

Reply:
I added the service around the function definition.

Reply:
The `option` should be indented more I think

Reply:
Seems to be fixed now.

Reply:
I... are you messing with me :joy:? I think you added one tab too many now.

Reply:
Nevermind, much better